.github/workflows/link-checker.yaml

```diff
@@ -1,61 +1,16 @@
 name: "Arc.codes Link Checker"
-
 on: [ pull_request ]
-
 defaults:
   run:
     shell: bash
-
 jobs:
-  build:
-    # Setup
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        node-version: [ 14.x ]
-        os: [ ubuntu-latest ]
-
-    # Go
+  linkchecker:
+    runs-on: ubuntu-latest
     steps:
-      - name: Check out repo
-        uses: actions/checkout@v2
-
-      - name: Set up Node.js
-        uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node-version }}
-
-      - name: Env
-        run: |
-          echo "Event name: ${{ github.event_name }}"
-          echo "Git ref:    ${{ github.ref }}"
-          echo "GH actor:   ${{ github.actor }}"
-          echo "SHA:        ${{ github.sha }}"
-          VER=`node --version`; echo "Node ver:   $VER"
-          VER=`npm --version`; echo "npm ver:    $VER"
-
-      - name: Install
-        run: npm install
-
-      - name: Vendor dist files
-        run: npm run dist
-
-      - name: Hydrate
-        run: npx arc hydrate
-        env:
-          CI: true
-
-      - name: check links
-        run: npm run link-checker
-        env:
-          CI: true
-
-      - name: Notify
```

Comment:
We don't want to notify discord of broken links?

Reply:
I don't think so. We don't notify Discord when other tests break. The notifications has been used for deploys and publishing. the end game things.
If the link checker job fails, the PR's checks are marked as "not successful" just like tests or the woke job.